packages/desktop-client/src/components/reports/ReportOptions.tsx

```diff
@@ -44,3 +44,92 @@ const intervalOptions = [
 { value: 5, description: 'Yearly', name: 5,
 ];
 */
+
+let uncategorizedCategory = {
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏do we have a type we could use to enforce all fields being correctly set here? If not - can we create it? Same for the others

```suggestion
let uncategorizedCategory: Something = {
```